### PR TITLE
add type of member (member or group or serviceAccount)

### PR DESCRIPTION
**Pull request by frajkallel**

### CODE REVIEW

**Comment by morgante** [comment]
@frajkallel Please run format and get the linters passing: https://github.com/terraform-google-modules/terraform-google-iam/blob/master/CONTRIBUTING.md

**Comment by frajkallel** [comment]
@morgante the linter is OK

File: modules/member_iam/variables.tf [hunk 1]
```diff
@@ -28,3 +28,9 @@ variable "project_roles" {
   description = "List of IAM roles"
   type        = list(string)
 }
+
+variable "prefix" {
+  description = "Prefix member or group or serviceaccount"
+  type        = string
```
Comment (morgante) [inline modules/member_iam/variables.tf:34]: We should provide a default (`serviceAccount`) here to keep it backwards-compatible.

File: examples/member_iam/variables.tf [hunk 2]
```diff
@@ -18,3 +18,7 @@ variable "project_id" {
   description = "Project id"
   type        = string
 }
+variable "prefix" {
```
Comment (morgante) [inline examples/member_iam/variables.tf:21]: No need to add this as a variable on the example, we can hardcode to `serviceAccount`.

File: test/fixtures/member-iam/main.tf [hunk 3]
```diff
@@ -17,4 +17,5 @@
 module "member_roles" {
   source     = "../../../examples/member_iam"
   project_id = var.project_id
+  prefix     = var.prefix
```
Comment (morgante) [inline test/fixtures/member-iam/main.tf:20]: No need to change fixtures.

File: examples/member_iam/variables.tf (hunk 2 above)
Comment (frajkallel) [reply examples/member_iam/variables.tf]: you have right ;)

File: modules/member_iam/variables.tf (hunk 1 above)
Comment (frajkallel) [reply modules/member_iam/variables.tf:34]: ok

**Comment by frajkallel** [comment]
Can you merge the PR @morgante ? i don't have write access

**Pull request merged**